docs/modules/getting-started/pages/persistence.adoc

```diff
@@ -0,0 +1,192 @@
+= Persist Data on Disk
+:description: This tutorial guides you on enabling persistence in your cluster.
+
+{description} Data in Hazelcast is usually stored in-memory (RAM) which
+makes it volatile; means that data is lost when too many members or the
+cluster is shut down. To recover data when the cluster is restarted, you
+can use Hazelcast's Persistence feature to persist data on disk.
+
+In this tutorial, you will:
+
+. Start a Hazelcast Enterprise cluster with three members for which persistence is enabled
+. Create a map with data in it
+. Shut down the cluster
+. Restart the cluster and see the persistence feature in action
+
+== Before You Begin
+
+To complete this tutorial, you need the following:
+
+[cols="1a,1a"]
+|===
+|Prerequisites|Useful resources
+
+|Trial license
+|https://trialrequest.hazelcast.com/[Hazelcast website]
+
+|Docker image for the full Hazelcast distribution
+|xref:get-started-enterprise.adoc[Install Hazelcast Enterprise]
+
+|Docker network with the name `hazelcast-network`
+|Use the `docker network create hazelcast-network` command 
+
+|===
+
+== Step 1. Start a Member with Persistence Enabled
+
+You'll start the first member in a cluster called `hello-world`. Run the following Docker command:
+
+[source,shell,subs="attributes+"]
+----
+docker run \
+    --name first-member --network hazelcast-network \
+    --rm \
+    -e HZ_CLUSTERNAME=hello-world \
+    -e HZ_LICENSEKEY=<your license key> \ <1>
```

Comment:
```suggestion
    -e HZ_LICENSEKEY=<your license key> \<1>
```

These spaces seem to cause errors with Docker. I get `invalid reference format`.